### ISSUE TITLE
Debug does not stop in subprojects

### ISSUE THREAD

**Issue by eiswind** [lucono/karma-test-explorer#47]
**Describe the Bug**

Create a new Angular (or even plain Typescript) project. Try to Debug. Works.
Move Angular project to a subfolder, create settings.json

```json
{
    
    "karmaTestExplorer.logLevel": "debug",
    "karmaTestExplorer.karmaLogLevel": "debug",
    "karmaTestExplorer.projects":["bulletin-board"]
}
```

Debug does not stop at any breakpoint anymore.

    Node Version: 16
    VS Code Version: 1.70.1
    Karma Version: 6.4
    Frameworks: (e.g. Angular, Jasmine, Mocha) Angular, Jasmine

**Comment by lucono** [comment]
I'm able to reproduce this. A workaround you can try is to set the `karmaTestExplorer.webRoot` setting to the project path as well:

```json
{
  "karmaTestExplorer.projects": ["bulletin-board"],
  "karmaTestExplorer.webRoot": "${workspaceFolder}/bulletin-board"
}
```
In a future release, the `webRoot` can be defaulted to the project path so that it should work without any special configuration.

**Comment by eiswind** [comment]
Thanks for the workaround. but would this work if  i have multiple projects in my workspace?

**Comment by lucono** [comment]
No, the workaround will only work for a single project in `karmaTestExplorer.projects`. I'll add a fix to the next release that should address it for multi-project setups.

**Comment by lucono** [comment]
@eiswind Try out the fix in the beta branch to confirm if it resolves the issue. You can use the [vsix](https://github.com/lucono/karma-test-explorer/suites/7946323741/artifacts/339196148) from the CI build, or build it yourself from the `beta` branch.

**Comment by eiswind** [comment]
found it a little surprising that I had to change karmaTestExplorer.projects to "karmaTestExplorer.projectWorkspaces".
Debugging works at my side w/o setting webroot.

**Comment by lucono** [comment]
For someone working with a single project which is an Angular workspace, the current naming of `karmaTestExplorer.projects` is not very intuitive as it could potentially be confused to mean the projects in the Angular workspace. The rename to `karmaTestExplorer.projectWorkspaces` is intended to avoid that confusion. The old naming will continue to work even with the rename, but the plan is to remove it at some point in a future update.

If you believe the new naming to be less intuitive, I'd be open to thoughts for a different name for that setting.

**Comment by eiswind** [comment]
I just found that it did not work without renaming. It complained that karma could not be found (which I think is because it did not check the folder)

**Comment by lucono** [comment]
Thanks, fixed in `beta` branch. Use updated [vsix](https://github.com/lucono/karma-test-explorer/suites/7983550481/artifacts/341709438) to confirm.

**Comment by eiswind** [comment]
I can confirm that both settings are working now.

**Comment by lucono** [comment]
Released in [v0.7.4](https://github.com/lucono/karma-test-explorer/blob/master/CHANGELOG.md#074---aug-27-2022)
